src/components/addCandidate/sagas.js

```diff
@@ -74,10 +74,10 @@ function* addCandidate(action) {
     /*
     *await ajax request to upload images and return a link
     */
-   if (!body.dateOfBirth || !picture) {
-       message.error(INCOMPLETE_FIELDS, WAIT_TIME);
-       yield put(actions.loadingaddCandidate(false));
-       return;
+    if (!body.dateOfBirth || !picture) {
```

Comment:
is body always set? what happens when body is null

Reply:
yes,body is always set, i set it with the payload of the action, it is a few lines above this. at the very least body is always set, then i just validate the body.